### PR TITLE
HDDS-6360. Create stack dump on acceptance test failure

### PR DESCRIPTION
**Pull request by adoroszlai**
## What changes were proposed in this pull request?

Sometimes acceptance test fails with timeout while writing data. Creating a stack dump for each container on test failure may be useful to debug the problem.

https://issues.apache.org/jira/browse/HDDS-6360

## How was this patch tested?

Induced timeout in Freon test, verified stack dumps were taken and included in the acceptance bundles.
https://github.com/adoroszlai/hadoop-ozone/actions/runs/1923597482

Regular CI:
https://github.com/adoroszlai/hadoop-ozone/actions/runs/1923899545

### CODE REVIEW

File: hadoop-ozone/dist/src/main/compose/testlib.sh [hunk 1]
```diff
@@ -201,6 +201,10 @@ execute_robot_test(){
 
   copy_daemon_logs
 
+  if [[ ${rc} -gt 0 ]] && [[ ${rc} -lt 250 ]]; then
```
Comment (smengcl) [inline hadoop-ozone/dist/src/main/compose/testlib.sh:204]: Just curious, why do we exclude exit codes >= 250?

**Review by smengcl — APPROVED** [review]
lgtm

File: hadoop-ozone/dist/src/main/compose/testlib.sh (hunk 1 above)
Comment (adoroszlai) [reply hadoop-ozone/dist/src/main/compose/testlib.sh:204]: http://robotframework.org/robotframework/3.2.1/RobotFrameworkUserGuide.html#return-codes

On second look, it should be `-le 250`.

**Comment by adoroszlai** [comment]
Thanks @smengcl for the review.

**Pull request merged**